pythonosc/osc_message_builder.py

```diff
@@ -78,6 +78,8 @@ def add_arg(self, arg_value, arg_type=None):
         arg_type = self.ARG_TYPE_TRUE
       elif builtin_type == builtins.bool and not arg_value:
         arg_type = self.ARG_TYPE_FALSE
+      else:
+        raise ValueError('Infered arg_value type is not supported')
```

Comment:
I disagree. `isinstance` will raise a `TypeError` only if the 2nd argument is not a type. Here, the 2nd argument is hardcoded with `str`, `bytes`, `int`, or `float` - all of them are proper python types.